### PR TITLE
Correction of Various Errors in the Tips and Tricks Documentation

### LINKED ISSUE #7833: Scrub references to Example File Generator from documentation
Issue overview
EnergyPlus Example File Generator has not been available for some time. Yet, the documentation continues to reference it, creating confusion.

### PR DESCRIPTION
**Pull request by RKStrand**
Pull request overview
---------------------
 - Fixes #7833 
 - A variety of problems were noted in Tips and Tricks.  Some of these were web sites that were no longer valid.  Others were glaring formatting issues.  In some cases, minor language edits were made (like where a chapter was totally blank).  While this was not a full clean-up of the document, it looks much better and all of the hyperlinks should now work and point to the correct address.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by Myoldmopar** [comment]
This seems like a good set of straightforward changes.  I will build locally but I don't have any problems.  And CI is happy of course, with one ERR file that is quite unrelated 😆 .

File: doc/tips-and-tricks-using-energyplus/src/building-geometry-shading-zone-model/using-multipliers-zone-and-or-window.tex [hunk 1]
```diff
@@ -187,11 +187,11 @@ \subsection{Background and Study using Multipliers}\label{background-and-study-u
...
-USA\_IL\_Chicago-OHare.Intl.AP.725300\_TMY3 & 0:51 & 1:34 & 9:37 \tabularnewline
-USA\_CA\_San.Francisco.Intl.AP.724940\_TMY3 & 0:50 & 1:34 & 9:59 \tabularnewline
-USA\_CO\_GOLDEN-NREL.724666\_TMY3 & 0:51 & 1:40 & 10:31 \tabularnewline
-USA\_FL\_Tampa.Intl.AP.722110\_TMY3 & 0:51 & 1:36 & 10:05 \tabularnewline
-USA\_VA\_Sterling-Washington.Dulles.Intl.AP.724030\_TMY3 & 0:51 & 1:36 & 9:48 \tabularnewline
+Chicago & 0:51 & 1:34 & 9:37 \tabularnewline
+San Francisco & 0:50 & 1:34 & 9:59 \tabularnewline
+Colorado & 0:51 & 1:40 & 10:31 \tabularnewline
+Tampa & 0:51 & 1:36 & 10:05 \tabularnewline
+Washington DC & 0:51 & 1:36 & 9:48 \tabularnewline
```
Comment (Myoldmopar) [inline doc/tips-and-tricks-using-energyplus/src/building-geometry-shading-zone-model/using-multipliers-zone-and-or-window.tex:194]: Tables look much nicer here.

File: doc/tips-and-tricks-using-energyplus/src/design-day-weather-data/weather-data-for-simulations.tex [hunk 2]
```diff
@@ -4,4 +4,4 @@ \section{Weather Data for Simulations}\label{weather-data-for-simulations}
 
 Drury B. Crawley. 1998. ``Which Weather Data Should You Use for Energy Simulations of Commercial Buildings?'' in ASHRAE Transactions, pp.~498-515, Vol. 104, Pt. 2. Atlanta: ASHRAE. (PDF 197 KB)
 
-PDF: \url{http://energyplus.gov/pdfs/bibliography/whichweatherdatashouldyouuseforenergysimulations.pdf}
+Click \href{https://energyplus.net/sites/all/modules/custom/weather/weather_files/whichweatherdatashouldyouuseforenergysimulations.pdf}{here} for the PDF of the above paper.
```
Comment (Myoldmopar) [inline doc/tips-and-tricks-using-energyplus/src/design-day-weather-data/weather-data-for-simulations.tex:7]: Works for me.

File: doc/tips-and-tricks-using-energyplus/src/hvac-sizing-equipment-simulation-and-controls/variable-refrigerant-flow-air-conditioner.tex [hunk 3]
```diff
@@ -1,6 +1,6 @@
 \section{Variable Refrigerant Flow Air Conditioner}\label{variable-refrigerant-flow-air-conditioner}
 
-\textbf{\emph{The EnergyPlus v7.0 release (October 2011) includes an initial model for VRF systems.~ See AirConditioner:VariableRefrigerantFlow and related objects.}}
+\textbf{\emph{Since its V7.0 release (October 2011), EnergyPlus has included a model for VRF systems.~ See AirConditioner:VariableRefrigerantFlow and related objects.}}
```
Comment (Myoldmopar) [inline doc/tips-and-tricks-using-energyplus/src/hvac-sizing-equipment-simulation-and-controls/variable-refrigerant-flow-air-conditioner.tex:3]: Better wording for sure.

File: doc/tips-and-tricks-using-energyplus/src/input/getting-data-from-window-program.tex [hunk 4]
```diff
@@ -20,7 +20,9 @@ \subsection{WINDOW Data File}\label{window-data-file}
 
 Please note that there is a bug in WINDOW 5 that causes two of the lines in the EnergyPlus data file to be joined. This bug is fixed in versions of Window 5.02 (and above). To be sure, you can check the data file for a line that looks like:
 
-GLAZING SYSTEM OPTICAL DATAAngle~~~~ 0~~~ 10~~~ 20~~~ 30~~~ 40~~~ 50~~~ 60~~~ 70~~~ 80~~~ 90 Hemis
+GLAZING SYSTEM OPTICAL DATA
```
Comment (Myoldmopar) [inline doc/tips-and-tricks-using-energyplus/src/input/getting-data-from-window-program.tex:23]: 👍

File: doc/tips-and-tricks-using-energyplus/src/utilities.tex [hunk 5]
```diff
@@ -1 +1,3 @@
 \chapter{Utilities}\label{utilities}
+
+EnergyPlus comes with a wide variety of pre- and post-processing that are found in the PreProcess and PostProcess folders in the main EnergyPlus install folder.  These utilities have been developed since the initial release of EnergyPlus and can be helpful during the creation of input data for EnergyPlus or interpreting its output.
```
Comment (Myoldmopar) [inline doc/tips-and-tricks-using-energyplus/src/utilities.tex:3]: Yeah that'll be better than nothing...literally!

**Review by Myoldmopar — APPROVED** [review]
Good to go.  Built locally, all changes look fine.

File: doc/tips-and-tricks-using-energyplus/src/utilities.tex (hunk 5 above)
Comment (RKStrand) [reply doc/tips-and-tricks-using-energyplus/src/utilities.tex:3]: I know! 😂😂😂 How long has this been like this?🤦‍♂️ Thanks for reviewing this!

**Pull request merged**